@facundominguez any idea why there is an error?
`stack_900` says: 

```
/home/circleci/project/tests/datacon/neg/NewTypes0.hs:1:1: error:
    Could not match the following expected errors with actual thrown errors:
    <Any Liquid error>
  |                
1 | {-@ LIQUID "--expect-any-error" @-}
  | ^              
                   
```

But in cabal and locally in my machine `NewTypes0` fails. Is there any check against expected error message?

--- a/tests/tests.cabal
+++ b/tests/tests.cabal
@@ -1101,7 +1101,7 @@ executable datacon-neg
                     , Data02
                     , Data02Lib
                     , Date
-                    , NewTypes0
+                    -- , NewTypes0
                     , NewTypes
 
     ghc-options:      -fplugin=LiquidHaskell -fkeep-going -O0

